templates/cf.yml

```diff
@@ -618,6 +639,8 @@ properties:
     tls:
       ca_cert: ~
     etcd:
+      ca_cert: (( .properties.etcd.ca_cert ))
+      require_ssl: (( .properties.etcd.require_ssl ))
```

Comment:
It sounds like some of the etcd properties you're copying over into the `loggregator.etcd` namespace, but stuff like `cluster` you're taking directly from the `etcd` namespace?